frontend/src/components/common/Resource/RestartButton.tsx

```diff
@@ -35,7 +36,7 @@ export function RestartButton(props: RestartButtonProps) {
 
   function applyFunc() {
     try {
-      const clonedItem = _.cloneDeep(item);
+      const clonedItem = _.cloneDeep(item) as Deployment;
```

Comment:
I think _.cloneDeep accepts a <type>

Reply:
yes it accepts a generic but it cannot accept Deployment. casting here is fine, clonedItem exists for just a couple of lines

Reply:
going to narrow down item type in props, the same way as in ScaleButton